pubspec.lock

```diff
@@ -35,14 +35,14 @@ packages:
       name: async
```

Comment:
I removed the hack for firebase messaging that we added as it's no longer needed